extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -60,6 +66,7 @@ public class OidcRecorder {
     private static final Logger LOG = Logger.getLogger(OidcRecorder.class);
 
     private static final Map<String, TenantConfigContext> dynamicTenantsConfig = new ConcurrentHashMap<>();
+    private static final Set<String> tenantsForWhichWeReportServerAvailable = ConcurrentHashMap.newKeySet();
```

Comment:
```suggestion
    private static final Set<String> tenantsExpectingServerAvailableEvents = ConcurrentHashMap.newKeySet();
```

It is just an internal var but not sure about `we` in its name :-)

Reply:
Well, someone has to do it... no seriously, I tried like 5 versions and your name is still better, thanks.